ebu_tt_live/config/node.py

```diff
@@ -192,6 +192,20 @@ def __init__(self, config, local_config):
         self._create_input(config)
         self._create_output(config)
 
+class Denester(ConsumerMixin, ProducerMixin, NodeBase):
```

Comment:
The class has to be referenced further down in [nodes_by_type](https://github.com/ebu/ebu-tt-live-toolkit/blob/bbc-python3-staging/ebu_tt_live/config/node.py#L352), in order to be used e.g. in a config file.

Reply:
Agreed, I think we have some updates to make to this pull request based on further work, and this is one of them.